rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -12,7 +12,7 @@ info:
   title: Query API
   description: >
     The Query API is defined to provide clients with the ability to
-    query for stored products.
+    query for stored IRM metadata.
```

Comment:
✏️ I think we can use the term `dataset` here even though it only returns IRM URIs right now.